sdk/cosmos/azure-cosmos-spark_3_2-12/pom.xml

```diff
@@ -98,7 +98,7 @@
     <dependency>
       <groupId>org.slf4j</groupId>
       <artifactId>slf4j-simple</artifactId>
-      <version>1.7.36</version> <!-- {x-version-update;org.slf4j:slf4j-simple;external_dependency} -->
+      <version>1.7.30</version> <!-- {x-version-update;cosmos_org.slf4j:slf4j-simple;external_dependency} -->
```

Comment:
Wouldn't this will cause conflicts with azure-cosmos shading?

Reply:
/azp run java - cosmos - ci